test/acceptance/AcceptanceTestSpec.scala

```diff
@@ -40,7 +40,7 @@ trait AcceptanceTestSpec extends FeatureSpec
   protected val ValidRequest = FakeRequest()
     .withHeaders(RequestHeaders.ACCEPT_HMRC_JSON_HEADER)
 
-  protected def idEndpoint(clientId: String, apiContext: String, apiVersion: String) =
+  protected def subscriptionEndpoint(clientId: String, apiContext: String, apiVersion: String) =
```

Comment:
could you rename it as `subscriptionFieldsEndpoint` please?

Reply:
Done